docs/content/en/docs/Development/_index.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Development
+weight: 30
+---
+
```

Comment:
Maybe add a short intro pargrapg with a reference back to getting_started here, saying that this assumes you've completed the steps from https://heir.dev/docs/getting_started/#building-from-source?

Reply:
In the intro paragraph, it might also be a good idea to explain what this page will cover and to link out to /bazel, /boilerplate and /ide. I know they'll be in the navigation, too, but people don't always explore those...

Oh, thinking about it, why don't we move the general parts of "Contributing to HEIR"  (everything above https://heir.dev/docs/contributing/#contributing-code-to-heir) to the https://heir.dev "landing page" and move the info about "Contributing code to HEIR"  here (Well, below ## Pre-Commit)? That way, heir.dev/development becomes a more-or-less "one stop shop" for all things development?

On that note: I think `research_with_heir` could also move into a development sub-page (and get a mention/link in the intro paragraph).